Sources/OpenCombine/CurrentValueSubject.swift

```diff
@@ -22,16 +24,19 @@ public final class CurrentValueSubject<Output, Failure: Error>: Subject {
 
     /// The value wrapped by this subject, published as a new element whenever it changes.
     public var value: Output {
-        didSet {
-            send(value)
+        get {
+            return _value
+        }
+        set {
+            send(newValue)
```

Comment:
The backing is not set here. Maybe, since we don’t synchronize anymore, we can revert this to just a stored property with a `didSet` like it was before?